Draft: Homepage and website skeleton (#4)
bbtr-org/bbtr: docs/index.md

# Accueil

L'objectif de ce site est d'aider les organisateurs de tournois de Magic: The Gathering, pour proposer la meilleure expérience possible aux joueurs. Ce site contient des ressources pour faciliter l'organisation de votre événement, en amont, le jour même et sur la durée.

## Qui sommes-nous ?

Les principaux contributeurs sur ce site sont Charles Gautier et Pierre Mourlanne.

Charles Gautier TODO TODO TODO

> Pierre Mourlanne a découvert Magic dans la cour du collège. Quelques années plus tard, il décide de passer une annee a ecumer les PTQs plutôt que de travailler les Maths et la Physique. Il ne se qualifie pas pour le Pro Tour, redouble son année de prépa, et décide donc de laisser Magic de côte. Il replonge en 2017, devient arbitre 2018. Maintenant arbitre de niveau 2, il organise des tournois a Poitiers avec l'association Damocles.

Reviewer: ```suggestion
Pierre Mourlanne a découvert Magic dans la cour du collège. Quelques années plus tard, il décide de passer une année à écumer les PTQs plutôt que de travailler les Maths et la Physique. Il ne se qualifie pas pour le Pro Tour, redouble son année de prépa, et décide donc de laisser Magic de côté. Il replonge en 2017, devient arbitre 2018. Maintenant arbitre de niveau 2, il organise des tournois à Poitiers avec l'association Damocles.
```
(lien sur le nom de l'asso ?)

Un de nos objectifs en créant ce site est de créer un espace de partage de connaissances. Si vous êtes vous-même organisateur de tournois, et que vous voulez contribuer sur le site, [contactez-nous](mailto:contact@bbtr.fr) !

## Pourquoi faire ce site ?

Nous voulons partager ce que nous avons appris en organisant des tournois. Des organisateurs locaux nous posent régulièrement des questions, nous demandent de l'aide, ce site est une référence de toutes nos connaissances.

> Notre objectif est de proposer la meilleure expérience possible aux joueurs. Nous pensons qu'il est possible de proposer des tournois de tres bonne qualite aux joueurs, sans se tuer a la tache, sans perdre d'argent, et en se faisant plaisir.

Reviewer: ```suggestion
Notre objectif est de proposer la meilleure expérience possible aux joueurs. Nous pensons qu'il est possible de proposer des tournois de tres bonne qualité aux joueurs, sans se tuer a la tâche, sans perdre d'argent, et en se faisant plaisir.
```

## Ça veut dire quoi "BBTR" ?

> "BBTR" signifie "Be Better". A un tournoi que nous organisions, un ami TO a rate le top huit de très peu. Alors qu'il s'en émouvait, quelqu'un lui a dit "Be Better". Notre ami n'avait qu'à être meilleur, et il aurait réussi.  

Reviewer: ```suggestion
"BBTR" signifie "Be Better". Lors d'un tournoi que nous organisions, un ami TO a raté le top huit de très peu. Alors qu'il s'en émouvait, quelqu'un lui a dit "Be Better". _Notre ami n'avait qu'à être meilleur, et il aurait réussi._
```

Les principaux contributeurs sur ce site sont Charles Gautier et Pierre Mourlanne.

> Charles a reçu ses premières cartes pour son 12ème anniversaire. Après une pause de plusieurs années, il se met à rejouer sous le bloc Theros, et prend goût à la compétition. Ayant un attrait particulier pour les formats Modern et Duel Commander, il décida avec un groupe d’amis de créer NainTré Magic et de proposer à sa communauté locale plusieurs événements par an, cela depuis 2017.

Reviewer: ```suggestion
Charles reçoit ses premières cartes pour son 12ème anniversaire. Après une pause de plusieurs années, il se met à rejouer sous le bloc Theros, et prend goût à la compétition. Ayant un attrait particulier pour les formats Modern et Duel Commander, il décide avec un groupe d’amis de créer NainTré Magic et de proposer à sa communauté locale plusieurs événements par an depuis 2017.
```

> Pierre a découvert Magic dans la cour du collège. Quelques années plus tard, il décide de passer une année à écumer les PTQs plutôt que de travailler les Maths et la Physique. Il ne se qualifie pas pour le Pro Tour, redouble son année de prépa, et décide donc de laisser Magic de côté. Il replonge en 2017, devient arbitre 2018. Maintenant arbitre de niveau 2, il organise des tournois à Poitiers avec [l'association Damoclès](https://discord.gg/T4F7Fy6).

Reviewer: ```suggestion
Pierre découvre Magic dans la cour du collège. Quelques années plus tard, il décide de passer une année à écumer les PTQs plutôt que de travailler les Maths et la Physique. Il ne se qualifie pas pour le Pro Tour, redouble son année de prépa, et décide donc de laisser Magic de côté. Il replonge en 2017, devient arbitre 2018. Maintenant arbitre de niveau 2, il organise des tournois à Poitiers avec [l'association Damoclès](https://discord.gg/T4F7Fy6).
```
